ruby-challenges.rb

```diff
@@ -75,4 +75,5 @@ def word_reader (string)
 # 1. input a method that will take in a string and decide if its a palindrome
 # 2. output a string that says whether the word is or isnt
 # 3.created the method and made if else statements with strings in them
-# 4.
\ No newline at end of file
+# 4.
```

Comment:
Great Work.  Remember to run your code before you submit.  Ruby #2 will not delete capital letters... 😉